### DISCUSSION TITLE
Can this Cc2652 Ali coordinator be used with Z2M?

### DISCUSSION THREAD

**Discussion by gert3d** [Koenkk/zigbee2mqtt#19661] [Devices]
https://nl.aliexpress.com/item/1005006185190083.html
And did anyone program it yet as a coordinator. Sorry if the answers  should be obvious, I have no idea

**Comment by lux73** [comment]
i cloud not say anything 'bout the mentioned Stick

but if you want a strong AND reliable zigbee Network you should consider to buy a recommend Coordinator from the official Website -> https://www.zigbee2mqtt.io/guide/adapters/#recommended
